File: apphub-frontend/src/modules/platform/monitoring/MonitoringEndpoints.js
import Endpoint from "common/js/dao/Endpoint";
import RequestMethod from "common/js/dao/RequestMethod";

export const MONITORING_PAGE = "/web/monitoring";

export const MONITORING_GET_FEATURES = new Endpoint(RequestMethod.GET, "/api/monitoring/features");
export const MONITORING_GET_FUNCTIONALITIES = new Endpoint(RequestMethod.GET, "/api/monitoring/features/{feature}/functionalities");
export const MONITORING_GET_SERVICES = new Endpoint(RequestMethod.GET, "/api/monitoring/{feature}/services");
export const MONITORING_GET_METRICS = new Endpoint(RequestMethod.GET, "/api/monitoring/metrics/{type}");

File: apphub-integration/src/main/java/com/github/saphyra/apphub/integration/framework/endpoints/MonitoringEndpoints.java
package com.github.saphyra.apphub.integration.framework.endpoints;

public class MonitoringEndpoints {
    public static final String MONITORING_PAGE = "/web/monitoring";

    public static final String MONITORING_GET_FEATURES = "/api/monitoring/features";
    public static final String MONITORING_GET_FUNCTIONALITIES = "/api/monitoring/features/{feature}/functionalities";
    public static final String MONITORING_GET_SERVICES = "/api/monitoring/{feature}/services";
    public static final String MONITORING_GET_METRICS = "/api/monitoring/metrics/{type}";
}
